README.md

```diff
@@ -54,6 +63,403 @@ npm install @dojo/shim
 
 The examples below are provided in TypeScript syntax. The package does work under JavaScript, but for clarity, the examples will only include one syntax.
 
+### Widget Routing
+
+Widgets are a fundamental concept for any Dojo 2 application and as such Dojo 2 Routing provides collection of components that integrate directly with existing widgets within an application.
```

Comment:
"Dojo 2 Routing provides collection of" -> "Dojo 2 Routing provides a collection of"